config/services/b2.toml

```diff
@@ -1,14 +1,18 @@
-# rustic config file to use opendal storage - in this case b2
+# rustic config file to use B2 storage via Apache OpenDAL
 [repository]
 repository = "opendal:b2" # just specify the opendal service here
 password = "mypassword"
+# or 
+# password-file = "/home/<username>/etc/secure/restic_passwd"
```

Comment:
maybe name it `rustic_passwd`? People using restic for sure will know that they can also use the restic password file here.